### ISSUE TITLE
Question: working with SpaCy

### ISSUE THREAD

**Issue by DonaldTsang** [TeamHG-Memex/eli5#260]
Does this library work with Textacy (SpaCy and Scikit-Learn)?

**Comment by lopuhin** [comment]
@DonaldTsang we never tested it with Textacy, so white-box (eli5.explain_prediction, eli5.explain_weights) is unlikely to work out of the box. Black-box (http://eli5.readthedocs.io/en/latest/blackbox/lime.html) approach should work though.
Which models from Textacy are you interested in?

**Comment by DonaldTsang** [comment]
The main focuses are:
- Text language classification (English, French, German, Spanish)
- tokenization, text cleaning, POS tagging and OOV detection
- Vectorization (internal SpaCy, Word2Vec and Fasttext with OOV)
- LDA, PLSI/PLSA, LSI/LSA and PCA/NMF topic-model clustering
- Sentiment analysis and context analysis
